metrics-server/src/chain.rs

```diff
@@ -24,16 +24,11 @@ impl BlockMetrics {
     fn register_staking(registry: &mut Registry, blockchain: Arc<RwLock<Blockchain>>) {
         let sub_registry = registry.sub_registry_with_prefix("staking");
 
-        let bc = blockchain.clone();
+        let bc = blockchain;
         let closure = NumericClosureMetric::new_gauge(Box::new(move || {
             bc.read().get_staking_contract().active_validators.len() as i64
         }));
         sub_registry.register("active_validators", "Number of active validators", closure);
-
-        let closure = NumericClosureMetric::new_gauge(Box::new(move || {
-            blockchain.read().get_staking_contract().parked_set.len() as i64
```

Comment:
Shouldn't we change it in favor of `inactive_validators`?

Reply:
Same reason as above